microsoft/testsuites/core/cpu.py

```diff
@@ -38,6 +72,52 @@ def _set_cpu_state(self, cpu_id: str, state: str, node: Node) -> bool:
         result = node.tools[Cat].read_from_file(file_path, force_run=True, sudo=True)
         return result == state
 
+    def _get_interrupt_data(self, node: Node) -> List[Interrupt]:
```

Comment:
This logic should be in class `Interrupt`. It's a better envelopes.

Reply:
The command can be called out of the `Interrupt`, but the parsing logic is in the class.

Reply:
What would be the best place to put this class? I am thinking of lisa/util ? Any suggestions?

Reply:
This class can be here. We can review it later, if it's reused. The `lisa/util` is for lisa only, it shouldn't have Distro specific things.